_data/utilities.yml

```diff
@@ -208,4 +208,4 @@ websites:
       url: https://voip.ms/
       img: voipms.png
       tfa: No
-      twitter: voipms
+      twitter: voipms
```

Comment:
As per our `.editorconfig`, it contains `insert_final_newline = true`.
Please respect that.